u8timeseries/models/baselines.py

```diff
@@ -28,7 +28,7 @@ def __str__(self):
 
     def fit(self, series: TimeSeries):
         super().fit(series)
-        self.mean_val = np.mean(series.values())
+        self.mean_val = np.mean(series.univariate_values())
```

Comment:
For univariate models, it would be nice if they print a user-friendly message when they are called with a multi-variate timeseries.
Perhaps one way to go would be to have 2 super-classes: `UnivariateForecastingModdel` and `MultivariateForecastingModel`, and the univariate one would do a check in its `fit()` method (which is called by all children), to ensure that the series it's called on is univariate, or alternatively that the user specified a column in the series. WDYT?